teralaunch/src/utils.js

```
@@ -0,0 +1,127 @@
+
+  return averageSpeed;
+}
+
+/**
+ * Format a time in seconds to a human-readable string.
+ * If the input is invalid, returns 'Calculating...'
+ * @param {number} seconds the time in seconds
+ * @returns {string} a human-readable string representation of the time
+ */
+export function formatTime(seconds) {
+  if (!isFinite(seconds) || seconds < 0) return "Calculating...";
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)